src/layout/PaymentDetails/PaymentDetailsComponent.tsx

```diff
@@ -20,6 +20,11 @@ export function PaymentDetailsComponent({ node }: IPaymentDetailsProps) {
   const mappedValues = FD.useMapping(mapping);
   const prevMappedValues = useRef<Record<string, unknown>>(mappedValues);
 
+  // refetch data on mount by invalidating cache as the first fetch is done by the formPrefetcher
+  useEffect(() => {
+    refetchOrderDetails();
+  });
+
```

Comment:
Doesn't this just fire on every render since there is no dependency array?

Reply:
Yep, once every render i guess. Should i handle it differently?

Reply:
The comment says it only refetches on mount, in that case you would want an empty dependency array?